charts/rancher-k3s-upgrader/0.4.0/values.yaml

```diff
@@ -0,0 +1,13 @@
+global:
+  cattle:
+    systemDefaultRegistry: ""
+
+systemUpgradeController:
+  image:
+    repository: rancher/system-upgrade-controller
+    tag: v0.10.0
+
+kubectl:
+  image:
+    repository: rancher/kubectl
+    tag: v1.23.3
```

Comment:
The previous chart has it at `v1.20.15`. Curious if it's updated for a specific reason or just a general "keeping up to date".

Regardless, there is `v1.23.7` published on dockerhub already, so any specific reason to not use it instead?

Reply:
I have no idea. I'm just bumping it and seeing if it breaks anything.

Guillerme told me not to use the 1.23.7 image he published as it is not likely reliable.